### DISCUSSION TITLE
Supporting dense layer _and_ Dependencies

### DISCUSSION THREAD

**Discussion by Carib3D** [OrcaSlicer/OrcaSlicer#1654] [Ideas]
I've just changed to OrcaSlicer from Superslicer. OS is great but I miss a few features, or maybe they are there but I can't find them. 

Supporting dense layer. In SS, this lays down a 50% infill between the set infill and the solid infill. In effect it allows you to have a lower infill % without compromising the top layer. Actually I have what I think is a better suggestion, which I have implemented 'manually' in a number of prints: I add a modifier with 4x the density of my standard infill, over a minimum of 4 layers. So if I have a 'general' infill of 5%, I would have a few layers at 20%. I have found this works well, especially where the top surface is domed.

Dependencies. In SS and PrusaSlicer dependencies can be set so that only specified filaments and processes show up for a given printer. As another suggestion, and this might be crazy talk, could another heading be added for Nozzles? So instead of (Printer-Filament-Process) headers, It becomes (Printer-Nozzle-Filament-Process). The Nozzle heading would take the Extruder section of the Printer and parts of the Quality & Support sections of the Process header that are relevant to nozzle size. If this could be implemented with dependencies, I believe it would simplify the workflow for those of us that use multiple printers and multiple nozzles.

**Comment by jeremytodd1** [comment]
Back in May I submitted an issue requesting for PrusaSlicer's "Extend Sparse Infill", which is a better version of SS's support dense layers feature.

https://github.com/SoftFever/OrcaSlicer/issues/1165

It has been added to the list of things to do, but work hasn't started on it yet.

Dependencies I fully agree with.  It'd be a great feature to have. The timing is kind of funny, actually. I just put in an issue for this right before seeing you posted a discussions about it:

https://github.com/SoftFever/OrcaSlicer/issues/1655

I agree with you and I hope both of these features are added, as they are very valuable.

**Reply by Carib3D** [reply]
I agree the sparse infill should get over the issue, although I wasn't aware that they had implemented that. Good on Prusa! Lets hope these features can appear on Orca soon!

**Reply by TheAnimatrix** [reply]
extend sparse infill imo is not better than SS version of support dense layer. You still need more infill than super slicer allows you to have

**Reply by julian-weinert** [reply]
I disagree. "extend sparse infill" does not produce the same good result as SS's support dense layers feature.
Especially when bridging large areas over sparse infill the super sparse bridge layer helps a lot. In Orca I usually need 4 top layers as with SS I can get away with... Let's call it 3.5, as the extra sparse bridging layer needs less material and print time.

<img width="850" height="537" alt="Screenshot 2025-12-25 at 23 16 34" src="https://github.com/user-attachments/assets/518e69ca-cabf-4e97-b3b5-aeb123d9c115" />
